c++.html

```diff
@@ -0,0 +1 @@
+only one
```

Comment:
change it to only two
```suggestion
only two
```